lib/Doctrine/ODM/MongoDB/DocumentManager.php

```diff
@@ -225,6 +220,21 @@ public function getClient(): Client
         return $this->client;
     }
 
+    public function getClientEncryption(): ClientEncryption
```

Comment:
We may want to mark this method as internal:
```suggestion
    /** @internal **/
    public function getClientEncryption(): ClientEncryption
```